frontend/audio.c

```diff
@@ -323,7 +323,7 @@ static size_t write_audio_16bit(audio_file *aufile, void *sample_buffer,
 
     if (aufile->channels == 6 && aufile->channelMask)
     {
-        for (i = 0; i < samples; i += aufile->channels)
+        for (i = 0; i+6 < samples; i += aufile->channels)
```

Comment:
it should be either `i + 6 <= samples` or `i + 5 < samples`

Reply:
Ok, I will modify it according to the suggestions. Thank you very much！